shared/oae/api/oae.api.util.js

```diff
@@ -1866,7 +1866,5 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
      *
      * @return {String}   Event(s) that can fire when the content of an input field changes
      */
-    var getFormChangeEventNames = exports.getFormChangeEventNames = function() {
-        return $('html').hasClass('ie-lt10') ? 'change keyup paste cut' : 'change input';
-    };
+    var getFormChangeEventNames = exports.getFormChangeEventNames = return "change input";
```

Comment:
@brecke 
>Was this what you did? 

These are the exact commands I used `git reset --hard <sha1-commit-id>` then `git push origin HEAD --force`

>But perhaps it would help if you told us what it was you tried to achieve in the first place?

I thought that I was working on the branch I created, but I discovered after committing and pushing my changes that I was working on the master. So, I wanted to reverse what I did.